### DISCUSSION TITLE
✅ I am unable to connect to ANY ssh tunnel after my initial setup

### DISCUSSION THREAD

**Discussion by sarman-tftsr** [isontheline/pro.webssh.net#570] [Q&A]
I installed and set up WebSSH yesterday. One complete, I tested my tunnels and they worked. Now, when I try I get the following error:
![00CB5692-CC63-4235-B1D0-D29BEF5F8932](https://user-images.githubusercontent.com/66151719/158809962-461c70ef-df6d-4ef1-a52a-cb02447b615b.png)

**Comment by isontheline** [comment]
@sarman-tftsr Please try : https://webssh.net/documentation/help/howtos/Tunnel/NEVPNErrorDomain/
